benchmarks/async_cases.py

```diff
@@ -4,7 +4,7 @@
 
 import taichi as ti
 
-sys.path.append(os.path.join(ti.core.get_repo_dir(), 'tests', 'python'))
+sys.path.append(os.path.join(sys.path[0], '../', 'tests', 'python/'))
```

Comment:
Can we briefly comment here on the path ;)? Before it was clear where it points to, now we depend on sys.path[0]?

Reply:
`ti.core.get_repo_dir()` now returns the directory`/home/username/.taichi/`,
and `sys.path[0]` points to the directory where the current file is located.

Reply:
maybe `__file__` is better? `sys.path` is a list of search path i believe.

Reply:
Thanks for the tip, when trying to use `__file__`, I found that the absolute path was missing
sys.path[0] = `/home/username/.../taichi_dir/benchmarks`
__file__ = `run.py`

Reply:
How about we use `os.path.dirname(os.path.realpath(__file__))`, which also serves the purpose